cms/settings.py

```diff
@@ -79,8 +80,10 @@
         "APP_DIRS": True,
         "OPTIONS": {
             "context_processors": {
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
```

Comment:
Where are these used?

Reply:
These were part of the Django upgrade

Reply:
I think they're used primarily by the admin interface

Reply:
If you remove them does the admin area break?

Reply:
So the only one we really need is `django.template.context_processors.request` to silence this warning:
```
WARNINGS:
?: (admin.W411) 'django.template.context_processors.request' must be enabled in DjangoTemplates (TEMPLATES) in order to use the admin navigation sidebar.```